tests/api.c

```diff
@@ -60713,7 +60729,10 @@ void ApiTest(void)
             printf(" %s\n", apitest_res_string(ret));
         }
         fflush(stdout);
-        AssertIntNE(ret, TEST_FAIL);
+        /* if return code is < 0 and not skipped then assert error */
+        Assert((ret > 0 || ret == TEST_SKIPPED),
```

Comment:
It looks like this change is uncovering a DTLS test issue with `test_wolfSSL_dtls_stateless_resume`. @julek-wolfssl will you review? Thanks

```
   705: test_wolfSSL_dtls_stateless_resume                  : invalid result (  0.01550)

ERROR - tests/api.c line 60733 failed with:
    expected: Test failed

    result:   ret -199

FAIL scripts/unit.test (exit status: 134)
```

Reply:
If I patch master with this change I can reproduce also.

Patch:

```
diff --git a/tests/api.c b/tests/api.c
index a521d3513..a2b3654b4 100644
--- a/tests/api.c
+++ b/tests/api.c
@@ -60713,7 +60713,11 @@ void ApiTest(void)
             printf(" %s\n", apitest_res_string(ret));
         }
         fflush(stdout);
-        AssertIntNE(ret, TEST_FAIL);
+        /* if return code is < 0 and not skipped then assert error */
+        Assert((ret > 0 || ret == TEST_SKIPPED),
+            ("Test failed\n"),
+            ("ret %d", ret));
+

         TestCleanup();
     }
```

```
./configure --enable-dtls --enable-dtls13 CFLAGS="-DWOLFSSL_DTLS_NO_HVR_ON_RESUME"
make check
...
./build-aux/test-driver: line 112: 43537 Abort trap: 6           "$@" >> "$log_file" 2>&1
FAIL: scripts/unit.test
```

```
   705: test_wolfSSL_dtls_stateless_resume                  : invalid result (  0.00449)

ERROR - tests/api.c line 60719 failed with:
    expected: Test failed

    result:   ret -199

FAIL scripts/unit.test (exit status: 134)
```